ttnn/cpp/ttnn/operations/data_movement/sharded/device/kernels/dataflow/reader_unary_stick_layout_sharded_blocks_interleaved_start_id.cpp

```diff
@@ -5,8 +5,25 @@
 #include <stdint.h>
 #include "dataflow_api.h"
 
-void kernel_main() {
+#define ENABLE_DEBUG 1
```

Comment:
can we move the print function to the data_movement/common folder ? 
Would be helpful for other kernels too :)